libsel4vm/src/arch/arm/vgic/vgicv2_defs.h

```diff
@@ -68,3 +68,10 @@
 #define GIC_DIST_SGI_CPU_TARGET_LIST_MASK       0xFF << GIC_DIST_SGI_CPU_TARGET_LIST_SHIFT
 
 #define GIC_DIST_SGI_INTID_MASK                 0xF
+
+/* Contants for getting registers from Device Tree */
+#define NUM_GIC_REGS        4
+#define GIC_DIST_ENTRY      0
```

Comment:
This should get a more verbose comment. How much of a device tree standard is this actually or is it just what everybody seems to use. I think it might be better to inline these values and not expose any constants. Then a comment there can explain the the magic values come from.  We can factor this out when these constants are to be somewhere. If we provide defines here, we names must be much more specific to avoid conflicts or get confusing.

Reply:
I found a link with the standard. Its not just what everyone seems to use.

Reply:
Can you rename this to e.g. `NUM_FDT_GIC_REGS` and `FDT_GIC_REG_IDX_<DIST|CPU|VCPU_CNTR|VCPU>`? So we have names that are a bit less ambiguous and clearly state this is device tree related